paasta_tools/cli/cmds/spark_run.py

```diff
@@ -292,6 +292,16 @@ def add_subparser(subparsers):
         default=CLUSTER_MANAGER_K8S,
     )
 
+    list_parser.add_argument(
+        "--enable-dra",
+        help=(
+            "Enable Dynamic Resource Allocation (DRA) for the Spark job as documented in https://yelpwiki.yelpcorp.com/display/AML/Spark+Dynamic+Resource+Allocation+%28DRA%29+Runbook. "
```

Comment:
Change to y/ link
Mention comment from user-perspective. They may not now Dynamic Resource Allocation.
Add a line on what it would do.

Reply:
Done